ceph.spec.in

```diff
@@ -266,12 +266,7 @@ BuildRequires:  libzbd-devel
 %if 0%{with jaeger}
 BuildRequires:  bison
 BuildRequires:  flex
-%if 0%{?fedora} || 0%{?rhel}
 BuildRequires:  json-devel
-%endif
-%if 0%{?suse_version}
-BuildRequires:  nlohmann_json-devel
-%endif
```

Comment:
> we dropped support for opensuse

i am confused. what does this imply?

Reply:
aah, I thought as we no longer build opensuse in shaman, we do not support it actively, don't we?

Reply:
> we do not support it actively, don't we?

to drop the suse support in `ceph.spec.in` would require more discussions and another PR. i'd suggest check the commit history of this very file for more context.

Reply:
gotcha, removed that commit. think we can wait on it.